### DISCUSSION TITLE
How to update to newest release?

### DISCUSSION THREAD

**Discussion by michael-schramm** [Leantime/leantime#814] [Q&A]
How do I update leantime to the newest release (ie. 2..2.0 --> 2.2.1)?

**Comment by marcelfolaron** [comment]
Hi,

To update Leantime just replace all the files in your current directory
with the new files. Config files and user files will not be affected. The
system will identify whether database updates are necessary and redirect
accordingly.

Sincerely,
Marcel

**Reply by michael-schramm** [reply]
Thanks!
It worked!

**Reply by eisteeverschluss** [reply]
> Hi, To update Leantime just replace all the files in your current directory with the new files. Config files and user files will not be affected. The system will identify whether database updates are necessary and redirect accordingly. Sincerely, Marcel
> […](#)
> On Mon, Jul 25, 2022 at 5:04 AM michael-schramm ***@***.***> wrote: How do I update leantime to the newest release (ie. 2..2.0 --> 2.2.1)? — Reply to this email directly, view it on GitHub <#814>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ALG4EFTS5IWLTLEW4QNUVUTVVZKA5ANCNFSM54RP7JQQ> . You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Hi, I tried this on a shared hosting. If i overwrite all files with the new ones, i only get blank page. how can i update leantime?

**Comment by marcelfolaron** [comment]
Please make sure you are running PHP8.
We dropped support for 7.4 with the latest version

**Comment by michael-schramm** [comment]
Hi,
I just updated from 2.4 beta to 2.4.2 but now I only get an error 500.
Using PHP 8.1, Apache server.

**Reply by michael-schramm** [reply]
I now tried using the updateLeantime.sh but I still get a 500 error.

**Reply by gloriafolaron** [reply]
Have you tried the most recent one with the same results? What errors are you seeing in your logs?
